File: Python/Clase-6/Clase_7/E3.py

print("Inserte la calificacion (del 1 al 10)")
a = input ("")
a = int(a)

if a >= 9 and a <=10 :
    print("A")
elif  a >= 8 and a <9:
    print("B")
elif  a >= 7 and a < 8:
    print("C")
elif  a >= 6 and a <7:
    print("D")
elif  a >= 0 and a <6:
    print("F") 
else:
    print("ERROR")
    

File: Java/clase1.0/E3.java
import java.util.Scanner;

public class E3 {
    public static void main(String[] args) {
          Scanner input = new Scanner (System.in); 
        
        System.out.println("Digite la cantidad de autos:  ");
        String n = input.nextLine();

        System.out.println("Digite el valor del auto:  ");
        String a = input.nextLine();

        int sueldo = (Integer.parseInt(n) * 150) + ((Integer.parseInt(a)/100)*5) + 1000;
        
        System.out.printf( "El total del sueldo es:  " + sueldo);
    }
}
